documentation/content/el/books/handbook/desktop/_index.adoc

```diff
@@ -610,7 +610,7 @@ endif::[]
 
 [source,shell]
 ....
-# pkg_add -r gqview
+# pkg install gqview
```

Comment:
This port was removed a few years ago, 

<https://www.freshports.org/graphics/gqview/>

Reply:
https://www.freshports.org/graphics/geeqie
fork still exists